spec/javascripts/jasmine_spec.rb

```diff
@@ -1,17 +1,13 @@
 # frozen_string_literal: true
 require 'English'
-require 'rake'
 
 # Run the jasmine tests by running the karma javascript test framework
 # The spec will fail if any jasmine tests fails.
 RSpec.describe "Jasmine" do
-  before do
-    Rails.application.load_tasks
-    Rake::Task["assets:clobber"].invoke
-    Rake::Task["assets:precompile"].invoke
-  end
-
   it "expects all jasmine tests to pass" do
+    # Ensure capybara is not using the remote browser
+    Capybara.using_driver(Capybara.javascript_driver) { Capybara.current_session.quit }
```

Comment:
This was the cause of the very flaky tests on this PR. Capybara leaves its session open, blocking any other drivers if they run later. Doing session cleanup one time here avoids recreating sessions for every JS feature spec.